src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -188,18 +189,49 @@ class PairedResult extends GraphContent {
      * @inheritDoc
      */
     resize(graph) {
+        let isPairedDataProper = true;
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+
+            //check if all region are there with respect to value
             if (graph.content.length > 1) {
-                hideAllRegions(graph.svg);
+                for (let i = 0; i < values.length; i++) {
+                    // eslint-disable-next-line max-depth
+                    for (const key in values[i]) {
```

Comment:
but need it  because values is an array for each value we have to check if the region is proper or not( for each value high , mid or low ) region is also there which we can not check for this constant(constants.PAIR_ITEM_TYPES).

Reply:
What are you trying to achieve with this? 

I pulled down your code and tried it myself: 

My `values[0]` is
![image](https://user-images.githubusercontent.com/4908886/62213336-4c532980-b368-11e9-8b54-554baf742733.png)

You are checking if high, mid and low are keys are present in the region, which obviously it wont.

![image](https://user-images.githubusercontent.com/4908886/62213599-e3b87c80-b368-11e9-8194-e62282f0fbc4.png)


Furthermore, I removed the nested for loop entirely and just have this
```javascript
 //check if all region are there with respect to value
            if (graph.content.length > 1) {
                if (
                    graph.config.isHideAllRegion === false
                ) {
                    checkRegionSame(graph.svg);
                    graph.config.isHideAllRegion = false;
                } else {
                    hideAllRegions(graph.svg);
                    graph.config.isHideAllRegion = true;
                }
            }
```
Yet, all your unit tests in PairedResult pass:
![image](https://user-images.githubusercontent.com/4908886/62213710-28dcae80-b369-11e9-8103-1c8a3b6a6d93.png)

Reply:
it's is because all the condition for region are proper i have not given those scenarios  where region is missing for any value, that is why all test case are passing

Reply:
what i am trying to achieve is i am checking for each value if region for each for each key in the value is there or not.

example - vlaue[0] - is an object having value for high, mid and low and i am checking that region is also there for all high mid and low, if for any value it is missing we will set the parameter  ispairedData proper as false and call hideAllRegions.

Reply:
So in short you are hiding the regions for high and low if there is not for high, low and mid? Is that a new requirement?

Reply:
it's very simple values=[
{ high:{},mid:{},low:{}}
]

regions={
high:{},
low:{}
}
so for low since there is no region we should hideAllRegions.